src/main/java/com/deloitte/elrr/aggregator/rules/ContextActivitiesTypeConstants.java

```diff
@@ -0,0 +1,17 @@
+package com.deloitte.elrr.aggregator.rules;
+
+import java.net.URI;
+
+public final class ContextActivitiesTypeConstants {
```

Comment:
It's generally referred to as `ContextActivityType` as opposed to `ContextActivitiesType`